catroid/src/main/res/values/strings.xml

```diff
@@ -1341,6 +1341,7 @@
     <string name="error_project_compatability">The program is not compatible with this version of the app.</string>
     <string name="error_outdated_version">This app version is outdated. Update now?</string>
     <string name="error_upload_project_with_default_name">You cannot upload a program with the name \'My first program\'. Please rename the program.</string>
+    <string name="error_upload_default_project">You cannot upload default program.</string>
```

Comment:
Please change this string, sounds a bit strange without article.
"You cannot upload the default program."

Reply:
I was expecting such a comment :) 
done.